### PR TITLE
Oreo: bump targets

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

File: sample_client/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 25
+    compileSdkVersion 26
     buildToolsVersion '25.0.0'
```
Comment (AndyScherzinger) [inline sample_client/build.gradle]: shoudl then always use 26.0.2

File: test_client/build.gradle [hunk 2]
```diff
@@ -20,7 +20,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 25
+    compileSdkVersion 26
     buildToolsVersion '25.0.0'
```
Comment (AndyScherzinger) [inline test_client/build.gradle]: same here, should always use 26.0.2

File: test_client/tests/build.gradle [hunk 3]
```diff
@@ -6,7 +6,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion 26
     buildToolsVersion "23.0.3"
```
Comment (AndyScherzinger) [inline test_client/tests/build.gradle]: and here: 26.0.2

File: sample_client/build.gradle (hunk 1 above)
Comment (tobiasKaminsky) [reply sample_client/build.gradle]: https://developer.android.com/studio/releases/gradle-plugin.html

> As of gradle 3.0 plugin: With this update, you no longer need to specify a version for the build tools—the plugin uses the minimum required version by default. So, you can now remove the android.buildToolsVersion property.

Do we want to remove it? Less config options is always nice, or?

**Comment by tobiasKaminsky** [comment]
Lint is complaining heavily for permissions, so I added them.
Those are the same as in our NC app.

File: AndroidManifest.xml [hunk 4]
```diff
@@ -29,8 +29,13 @@
     android:versionName="1.0.13" >
 
     <uses-sdk
-        android:minSdkVersion="11"
-        android:targetSdkVersion="24" />
+        android:minSdkVersion="14"
+        android:targetSdkVersion="26"/>
+
+    <uses-permission android:name="android.permission.AUTHENTICATE_ACCOUNTS"/>
```
Comment (mario) [inline AndroidManifest.xml]: Please put android:maxSdkVersion="22" for all of them.

**Comment by tobiasKaminsky** [comment]
Ready for merge @mario

**Comment by mario** [comment]
@tobiasKaminsky thanks. Please do the maxsdk dance in the app repo too :)

**Pull request merged**